sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Create.java

```diff
@@ -586,9 +586,10 @@ public T apply(TimestampedValue<T> input) {
       throws CannotProvideCoderException {
     checkArgument(
         !Iterables.isEmpty(elems),
-        "Elements must be provided to construct the default Create Coder. To Create an empty "
-            + "PCollection, either call Create.empty(Coder), or call 'withCoder(Coder)' on the "
-            + "result PTransform");
+        "This Create class cannot deduce a Coder because it has zero " +
```

Comment:
This isn't actually correct. The Create instance is what can't produce a default coder; if it were given elements, it may be able to.

Perhaps "Could not construct the default create Coder because this Create has no elements...."